Address AD review comments from Ben Kaduk

dthaler opened the pull request:
All but two AD comments are addressed in this PR

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

dthaler committed:
Address AD review comments from Ben Kaduk

All but two comments are addressed in this PR

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

dthaler commented:
@kaduk addressing most of your comments in this PR

mingpeiwk commented on draft-ietf-teep-architecture.md:
```diff
@@ -1327,7 +1339,11 @@ from being disclosed to the TAM that distributes them.  In such a scenario,
 must be some means of provisioning the decryption key into the TEE and/or some
 means of the Trusted Component Signer securely learning a public key of the TEE that it can use to
-encrypt.  One way to do this is for the Trusted Component Signer to run its own TAM so that it can
+encrypt.  The Trusted Component Signer cannot necessarily even trust the
```
Drop a space here: "encrypt. The..."

dthaler replied:
Done

mingpeiwk reviewed (approved):
Thanks for the changes; look good.

dthaler committed:
Changes for Ben's comments originally on transport doc

As Ben stated in his review, some comments are better addressed
in a different doc, so these are applied here in the arch doc.

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

dthaler committed:
Drop space

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

dthaler commented:
Merging per Ming's approval
